src/coreclr/build-runtime.sh

```diff
@@ -191,11 +191,11 @@ __BuildJit=1
 __BuildAllJits=1
 __BuildRuntime=1
 
-source "$__ProjectRoot"/_build-commons.sh
+#if [[ "${__BuildArch}" != "${__HostArch}" && "${__BuildOS}" == "OSX" ]]; then
+#    __CrossBuild=1
+#fi
```

Comment:
@janvorli Is this intentional?  Should it just be removed?

Reply:
I am sorry, I've missed it before pushing the commit, it should be removed.